app/schemas/models/classroom.schema.coffee

```diff
@@ -11,6 +11,13 @@ _.extend ClassroomSchema.properties,
   codeCamel: c.shortString(title: "UpperCamelCase version of code for display purposes")
   aceConfig:
     language: {type: 'string', 'enum': ['python', 'javascript']}
+  averageStudentExp: { type: 'string' }
+  ageRangeMin: { type: 'string' }
+  ageRangeMax: { type: 'string' }
```

Comment:
That would probably be awkward for the "Younger than 6" and "Older than 18" brackets.